core/trino-main/src/main/java/io/trino/sql/gen/columnar/CallColumnarFilterGenerator.java

```diff
@@ -355,7 +353,7 @@ else if (argumentExpression instanceof ConstantExpression) {
 
         InvocationConvention invocationConvention = new InvocationConvention(
                 builder.build(),
-                FAIL_ON_NULL,
+                resolvedFunction.functionNullability().isReturnNullable() ? DEFAULT_ON_NULL : FAIL_ON_NULL,
```

Comment:
I'm using `DEFAULT_ON_NULL` to adapt any nulls to `false` for nullable return functions. This is because we just treat NULL positions as false right now in columnar filter evaluation.

Reply:
Ya. Was just asking that because if the function returned say `long`, it would return a zero.